pkg/operator/v411_00_assets/bindata.go

```diff
@@ -4,6 +4,9 @@
 // bindata/v4.1.0/config/defaultconfig.yaml
 // bindata/v4.1.0/kube-controller-manager/cluster-policy-controller-cm.yaml
 // bindata/v4.1.0/kube-controller-manager/cm.yaml
+// bindata/v4.1.0/kube-controller-manager/csr_approver_clusterrole.yaml
```

Comment:
move autogenerated changes to a separate commit, please

Reply:
Done.